iot-hub/Samples/device/TwinSample/Program.cs

```diff
@@ -24,6 +24,11 @@ public static async Task<int> Main(string[] args)
                 .WithParsed(parsedParams =>
                 {
                     parameters = parsedParams;
+                    if (parameters.TransportType == TransportType.Http1)
+                    {
+                        Console.WriteLine("Twin update notifications not supported over HTTP.");

```

Comment:
Twin ~update notifications~ operations are not supported over HTTP

Reply:
updated.